db/migrate/20230816201738_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :post_counter
+      # t.datetime :created_at
+      # t.datetime :updated_at
```

Comment:
- [ ] Kindly make sure to remove all commented code from your code base, it makes your code messy and hard to read